core_data_modules/cleaners/codes/kenya_codes.py

```diff
@@ -0,0 +1,639 @@
+        BELGUT: KERICHO,
+        SIGOWET_SION: KERICHO,
+        SOTIK: BOMET,
+        CHEPALUNGU: BOMET,
+        BOMET_EAST: BOMET,
+        BOMET_CENTRAL: BOMET,
+        KONOIN: BOMET,
+        LUGARI: KAKAMEGA,
+        LIKUYANI: KAKAMEGA,
+        MALAVA: KAKAMEGA,
+        LURAMBI: KAKAMEGA,
+        NAVAKHOLO: KAKAMEGA,
+        MUMIAS_WEST: KAKAMEGA,
+        MUMIAS_EAST: KAKAMEGA,
+        MATUNGU: KAKAMEGA,
+        BUTERE: KAKAMEGA,
+        KHWISERO: KAKAMEGA,
+        SHINYALU: KAKAMEGA,
+        IKOLOMANI: KAKAMEGA,
+        VIHIGA: VIHIGA,
+        SABATIA: VIHIGA,
+        HAMISI: VIHIGA,
+        LUANDA: VIHIGA,
+        EMUHAYA: VIHIGA,
+        TESO_NORTH: BUSIA,
+        TESO_SOUTH: BUSIA,
+        NAMBALE: BUSIA,
+        MATAYOS: BUSIA,
+        BUTULA: BUSIA,
+        FUNYULA: BUSIA,
+        BUDALANGI: BUSIA,
+        MOUNT_ELGON: BUNGOMA,
+        SIRISIA: BUNGOMA,
+        KABUCHAI: BUNGOMA,
+        BUMULA: BUNGOMA,
+        KANDUYI: BUNGOMA,
+        WEBUYE_EAST: BUNGOMA,
+        WEBUYE_WEST: BUNGOMA,
+        KIMILILI: BUNGOMA,
+        TONGAREN: BUNGOMA,
+        UGENYA: SIAYA,
+        UGUNJA: SIAYA,
+        ALEGO_USONGA: SIAYA,
+        GEM: SIAYA,
+        BONDO: SIAYA,
+        RARIEDA: SIAYA,
+        KISUMU_EAST: KISUMU,
+        KISUMU_WEST: KISUMU,
+        KISUMU_CENTRAL: KISUMU,
+        SEME: KISUMU,
+        NYANDO: KISUMU,
+        MUHORONI: KISUMU,
+        NYAKACH: KISUMU,
+        KASIPUL: HOMA_BAY,
+        KASIPUL_KABONDO : HOMA_BAY,
+        KARACHUONYO: HOMA_BAY,
+        RANGWE: HOMA_BAY,
+        HOMA_BAY_TOWN: HOMA_BAY,
+        NDHIWA: HOMA_BAY,
+        MBITA: HOMA_BAY,
+        SUBA: HOMA_BAY,
+        RONGO: MIGORI,
+        AWENDO: MIGORI,
+        SUNA_EAST: MIGORI,
+        SUNA_WEST: MIGORI,
+        URIRI: MIGORI,
+        NYATIKE: MIGORI,
+        KURIA_WEST: MIGORI,
+        KURIA_EAST: MIGORI,
+        BONCHARI: KISII,
+        BONCHARI: KISII,
+        SOUTH_MUGIRANGO: KISII,
+        BOMACHOGE_BORABU: KISII,
+        BOBASI: KISII,
+        BOMACHOGE_CHACHE: KISII,
+        NYARIBARI_MASABA: KISII,
+        NYARIBARI_CHACHE: KISII,
+        KITUTU_CHACHE_NORTH: KISII,
+        KITITU_CHACHE_SOUTH: KISII,
+        KITUTU_MASABA: NYAMIRA,
+        WEST_MUGIRANGO: NYAMIRA,
+        NORTH_MUGIRANGO: NYAMIRA,
+        BORABU: NYAMIRA,
+        WESTLANDS: NAIROBI,
+        DAGORETTI_NORTH: NAIROBI,
+        DAGORETTI_SOUTH: NAIROBI,
+        LANG_ATA: NAIROBI,
+        KIBRA: NAIROBI,
+        ROYSAMBU: NAIROBI,
+        KASARANI: NAIROBI,
+        RUARAKA: NAIROBI,
+        EMBAKASI_SOUTH: NAIROBI,
+        EMBAKASI_NORTH: NAIROBI,
+        EMBAKASI_CENTRAL: NAIROBI,
+        EMBAKASI_EAST: NAIROBI,
+        EMBAKASI_WEST: NAIROBI,
+        MAKADARA: NAIROBI,
+        KAMUKUNJI: NAIROBI,
+        STAREHE: NAIROBI,
+        MATHARE: NAIROBI
+    }
+
+    CONSTITUENCIES = list(CONSTITUENCY_TO_COUNTY_MAP.keys())
+
+    COUNTIES = list(CONSTITUENCY_TO_COUNTY_MAP.values())
```

Comment:
This will give you duplicates in the COUNTIES list. Suggest list(set(CONSTITUENCY....values()))